doc/how-to-use.md

```diff
@@ -0,0 +1,400 @@
+## How to use multus-cni?
+
+### Prerequisite
+
+Kubelet must be configured to run with the CNI network plugin. Edit `/etc/kubernetes/kubelet` file and add `--network-plugin=cni` flags in `KUBELET\_OPTS `as shown below:
+
+```
+KUBELET_OPTS="...
+--network-plugin-dir=/etc/cni/net.d
+--network-plugin=cni
+"
+```
+
+### Install multus
```

Comment:
This section has two bullets, "copy multus binary" and "use daemonset". I think it would be clearer if the text associated with each bullet were indented. Not sure if that is possible, the associated sentence may need an indented bullet '   -', but your call. Just a suggestion.

Reply:
Add indent.